website/project/model.py

```diff
@@ -2217,6 +2217,7 @@ def register_node(self, schema, auth, data, parent=None):
         registered.logs = self.logs
         registered.tags = self.tags
         registered.piwik_site_id = None
+        registered.primary_institution = self.primary_institution
```

Comment:
Would be good to have a regression test for this somewhere in this test class: https://github.com/CenterForOpenScience/osf.io/blob/8b1e7498253343db3660802a7be82610147c4374/tests/test_models.py#L3776